_posts/2019-10-08-lambda-to-openfaas.md

```diff
@@ -25,7 +25,7 @@ Let's see what it takes to upgrade your AWS Lambda functions to OpenFaaS and Ser
 ## Prerequisites
 
```

Comment:
I'd suggest putting yourself in as a co author / editor like I've done on some of the collaborative posts we've worked on. That way people know someone is still maintaining this article.

You may also want to add a `> Last Updated: 2023.... ` up top.